maartenleenders/whoiswhere
# Introduce styled components

## Summary
* Introduces styled components to be used in the table of presence.

## Relevant technical decisions
* Adds dependencies `styled-components` and `@yoast/style-guide`.
* Adds `svg` images to signify different Yoast buildings.

## Testing instructions
* Run `yarn install` in the `root` and in the `client` folder
* Run `yarn start` in the `client` folder
* Click on things around and check styling things up did not break them
